rollup.config.js

```diff
@@ -69,7 +69,8 @@ export default [
             getUsername: 'src/selectors/userSelectors.js',
             errorTheme: 'src/themes/errorTheme.js',
             linnTheme: 'src/themes/linnTheme.js',
-            fetchError: 'src/reducers/reducerfactories/fetchErrorReducerFactory.js'
+            fetchError: 'src/reducers/reducerfactories/fetchErrorReducerFactory.js',
+            printCss: '/printStyles.css'
         },
```

Comment:
I would remove the css file from this list. All this part of the rollup config is doing is exposing all our components as ES modules (as opposed to  exports withing a common js bundle). I wouldn't imagine rollup will let you convert a css file to an ES6 module so this line will probably fail? Either way we don't currently tend to use the es modules imports anyway.

Reply:
Oops just realised you closed this PR 🙃